MariaDB/db/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM verit/klaros-mariadb_db:4.12.6
+FROM verit/klaros-mariadb_db:4.12.7
```

Comment:
The tag "4.12.7" has not yet been released in Docker Hub.
That also counts for all other images.